scenarios/Makefile

```diff
@@ -14,6 +23,14 @@ CURRENT_USER       := $(shell id -u):$(shell id -g)
 # Set to devel to use in-development versions of components
 UP4_ENV ?= stable
 
+# Build sdfabric-onos image as part of the deps step
+BUILD_SDFABRIC ?= true
```

Comment:
I don't like this variable to define if building the sdfabric-onos or not. But I haven't found an easier way to define it as part of the env file, or to decide upon the presence of the `_VERSION` variable in the env file.

Reply:
Can we depend on `ONOS_IMAGE` being empty like we discussed in the sync up document?

Reply:
It's a bit tricky, because ONOS_IMAGE is part of the env file, so doing checks on the variable status as part of the makefile requires to first export the env file, and then in the same shell do the if. If that helps, I can provide an example.

Maybe there's a straightforward way to do that, but I haven't found one.

Reply:
> Can we depend on `ONOS_IMAGE` being empty like we discussed in the sync up document?

Yeah this sounds like a good idea. It might also eliminate the need to replace ONOS_IMAGE variable below.

Reply:
To better clarify, it's doable (I had first tried in that way), but it makes the makefile quite messy and difficult to read (and probably to maintain).

Reply:
How about using a special value `ONOS_IMAGE=sdfabric-onos-local` so you don't have to write the .env file in the make target and the same special value can be used as the actual image tag?

Reply:
Let me provide that implementation on a separate PR, so we can easily compare the two approaches. I'll push it tomorrow.